src/js/handleGame.js

```diff
@@ -0,0 +1,164 @@
+import createGameBoards from "./boardView.js";
+import {UltimateBoard} from "./Boards/UltimateBoard.js";
+import {HumenPlayer} from "./Players/HumenPlayer.js";
+import {ComputerPlayer} from "./Players/ComputerPlayer.js";
+import { Board } from "./Boards/Board.js";
+
+const   boardSize = 3,
+        elementsID = {gameStatus: 'gameStatus',
+                    humPlayersign: 'humPlayerSign',
+                    compPlayersign: 'compPlayerSign'};
+let humPlayer,
+    compPlayer,
+    currPlayer,
+    bigBoard,
+    scoreBrd;
+
+
+export function startGame(mainBoardElem, scoreElem, gameInfoElem){
+    initGameBoards(mainBoardElem, scoreElem);
+    initPlayers();
+    initGameInfo(gameInfoElem);
+    updateCurrPlayer();
+}
+
+function initGameBoards(mainBoardElem, scoreElem){
+    // Creating all game boards
+    const brds = createGameBoards(boardSize);
+
+    mainBoardElem.appendChild(brds.mainBrd.tblElem);
+    scoreElem.appendChild(brds.scoreBrd.tblElem);
+
+    bigBoard = new UltimateBoard("brd-0", boardSize, brds.mainBrd.tblElem, brds.mainBrd.cellsArr);
+    scoreBrd = new Board("scrBrd", boardSize, brds.scoreBrd.tblElem, brds.scoreBrd.cellsArr);
+
+    addClickEventCells();
+}
+
+function initPlayers(plyr1, plyr2){
+    humPlayer = new HumenPlayer("You", "X");
+    compPlayer = new ComputerPlayer("Comp", "O");
+}
+
+function initGameInfo(gameInfoElem){
+    // const elem =  document.createElement("p");
+    //.setAttribute();
+
+    $(`#${elementsID.humPlayersign}`).html(`<p>Player 1: ${humPlayer.sign}</p>`);
+    $(`#${elementsID.compPlayersign}`).html(`<p>Player 2: ${compPlayer.sign}</p>`);
+}
+
+function updateCurrPlayer(){
+    if (currPlayer === humPlayer){
+        currPlayer = compPlayer;
+    } else {
+        currPlayer = humPlayer;
+    }
```

Comment:
Neat: use an array of size 2 for 2 players, and move to next player by increasing the index and modulu the array size (endless cyclic traversal)